docs/Makefile

```diff
@@ -4,14 +4,13 @@
 # You can set these variables from the command line.
```

Comment:
I think there's an easier way with less code duplication of this file to accomplish what you're trying to do.

The incoherence of having a symlink in one doc, and not having it in the others bothers me a little bit. So here's my suggestion:

IMHO this file you can leave it as it was before, and on the `tox.ini` file, you can do the following

```ini
[testenv:zenko-operations]
description = Render Zenko Operations Guide
skip_install = true
deps =
    -r{toxinidir}/docs/requirements.txt
commands =
    make -C docs/ {posargs:html} SPHINXOPTS="-j 4 -n -W" SPHINXPROJ="Zenko-Operations" SOURCEDIR="Operation-Architecture"
whitelist_externals =
    make

[testenv:zenko-setup]
description = Render setup guide
skip_install = true
deps =
    -r{toxinidir}/docs/requirements.txt
commands =
    make -C docs/ {posargs:html} SPHINXOPTS="-j 4 -n -W" SPHINXPROJ="Installation" SOURCEDIR="Installation"
whitelist_externals =
    make

[testenv:zenko-reference]
description = Render reference guide
skip_install = true
deps =
    -r{toxinidir}/docs/requirements.txt
commands =
    make -C docs/ {posargs:html} SPHINXOPTS="-j 4 -n -W" SPHINXPROJ="Reference" SOURCEDIR="Reference"
whitelist_externals =
make
```

Note that now I'm only ONE makefile, and I'm now customizing the variables inside the makefile. So when you will modify it in the future, you don't need to make it more than one.

Reply:
If you apply the requested change, you can delete the `Makefiles` inside each doc, and only leave one at the upper level

Reply:
I tested your proposed changes and the project stopped building. 

I don't doubt that what you're proposing is better, but I need you to test it on your end. 

Open an Improvement ticket?

Reply:
Sure thing!